src/components/Login/Login.tsx

```diff
@@ -16,37 +16,23 @@ const Login = (): JSX.Element => {
     <LoginStyled>
       <Formik
         initialValues={initialValues}
-        validate={(values) => {
-          const errors: ErrorsHandler = {};
-          if (!values.username) {
-            errors.username = "Required username";
-          }
-
-          if (!values.password) {
-            errors.password = "Required password";
-          }
-
-          return errors;
-        }}
+        validationSchema={validateLoginSchema}
         onSubmit={(values, { resetForm }) => {
-          setTimeout(() => {
-            console.log(values);
-            resetForm();
-          }, 2000);
+          resetForm();
         }}
       >
-        {({ errors, isSubmitting }) => (
-          <Form autoComplete="off" className="form">
+        {({ errors, isValid }) => (
+          <Form autoComplete="off" className="form" noValidate>
             <div className="input-container">
```

Comment:
Move all this <Form>....</Form> code into a component, and use the formikContext there.

This way you can remove the line from above 
`{({ errors, isValid }) => (`

And in the new component:
 const {errors, isValid} = useFormikContext<formSchemaType>()

Reply:
Just moved it all on the las pull request.